src/main/java/com/thealgorithms/others/SpiralMatrixII.java

```diff
@@ -0,0 +1,49 @@
+package com.thealgorithms.others;
+
+import java.util.Arrays;
+
+public class SpiralMatrixII {
```

Comment:
Why not to make a _proper utility class_ out of this?
```suggestion
public final class SpiralMatrixII {
    private SpiralMatrixII() {
    }
```

Reply:
Will do it